Cargo.toml

```diff
@@ -9,14 +9,15 @@ edition = "2021"
 anyhow = "1.0"
 hex = { version = "0.4", features = ["serde"] }
 num = "0.4"
-plonky2 = { git = "https://github.com/mir-protocol/plonky2", rev = "d527073416dd8cbc04457b8cb856b2fec3fa0e6c" }
-plonky2_ecdsa = { git = "https://github.com/mir-protocol/plonky2", rev = "d527073416dd8cbc04457b8cb856b2fec3fa0e6c", optional = true }
+plonky2 = { git = "https://github.com/InternetMaximalism/plonky2", rev = "4cb0b48df1d227d5461a4c28ed025aaea64e2e62" }
```

Comment:
Please add `default-features = false`

Reply:
I added `default-features = false`, but it did not seem to work without the `std` feature, so I added it by default (L.20).